Improved home page documentation (#1047)
guardicore/monkey: docs/content/_index.md

## What is Guardicore Infection Monkey?

> The Infection Monkey is an open source breach and attack simulation tool for testing a data center's resiliency to perimeter breaches and internal server infection.

Reviewer: ```suggestion
The Infection Monkey is an open-source breach and attack simulation tool for testing a data center's resiliency to perimeter breaches and internal server infection.
```

> Infection Monkey will help you test implemented security solutions and will provide visibility of the internal network through the eyes of an attacker.

Reviewer: ```suggestion
Infection Monkey will help you validate your existing security solutions and will provide a view of your network from an attacker's perspective.
```

Author: Fixed, but left the word _internal_, because for monkey to work effectively it does need to be  in an internal network.

> Infection Monkey will help you validate existing security solutions and will provide a view of the internal your network from an attacker's perspective.

Reviewer: ```suggestion
Infection Monkey will help you validate existing security solutions and will provide a view of your internal network from an attacker's perspective.
```

Infection Monkey is free and can be downloaded from [our homepage](https://infectionmonkey.com/).

![Infection Monkey Documentation Hub Logo](/images/monkey-teacher.svg?height=400px "Infection Monkey Documentation Hub Logo")

## How it works

> Architecturally Infection Monkey tool is comprised of two parts:

Reviewer: ```suggestion
Architecturally, Infection Monkey is comprised of two components:
```

> * Monkey Agent (Monkey for short) - a safe, worm like binary program which scans, spreads and simulates attack techniques on the **local network**.

Reviewer: ```suggestion
* Monkey Agent (Monkey for short) - a safe, worm-like binary program which scans, propagates and simulates attack techniques on the **local network**.
```

> * Monkey Island Server (Island for short) - a C&C web server which serves GUI for users and interacts with Monkey Agents.

Reviewer: ```suggestion
* Monkey Island Server (Island for short) - a C&C web server which provides a GUI for users and interacts with the Monkey Agents.
```

> User runs Monkey Agent on the Island server machine or distributes Monkey Agent binaries on the network manually. Based on 

Reviewer: Using "User" as a proper noun or pronoun sounds strange. "The user can run" or something similar reads better.

> the configuration parameters, Monkey Agents scan, propagate and simulate attackers behavior on the local network. All of the 

Reviewer: ```suggestion
the configuration parameters, Monkey Agents scan, propagate and simulate an attacker's behavior on the local network. All of the 
```

Reviewer: This has to be `an attacker's` or `attackers'`, but `attacker's` is incorrect.

Author: Fixed

> User runs Monkey Agent on the Island server machine or distributes Monkey Agent binaries on the network manually. Based on 
> the configuration parameters, Monkey Agents scan, propagate and simulate attackers behaviour on the local network. All of the 
> information gathered about the network is aggregated in the Island Server and displayed once all Monkey Agents are finished.

Reviewer: ```suggestion
The user can run the Monkey Agent on the Island server machine or distribute Monkey Agent binaries on the network manually. Based on 
the configuration parameters, Monkey Agents scan, propagate and simulate attackers behavior on the local network. All of the 
information gathered about the network is aggregated by the Island Server and displayed once all Monkey Agents are finished.
```

Author: Not sure what the first part solves.

Reviewer: It flows a little better and is more consistent with some of the edits that @MarketingYeti has been making.

Author: New version is longer. Also, we're describing a workflow, and the word `can` is unnecessary, since there's no case where you deploy island but don't run monkey. Changed regardles

## Results

> Results of running Monkey Agents are:

Reviewer: ```suggestion
The results of running Monkey Agents are:
```

>  - A map which displays how much of the network attacker can see, services accessible and potential propagation routes.

Reviewer: ```suggestion
 - A map which displays how much of the network an attacker can see, what services are accessible and potential propagation routes.
```

>  - Security report, which displays security issues Monkey Agents found and/or exploited.

Reviewer: ```suggestion
 -  A security report, which displays security issues that Monkey Agents discovered and/or exploited.
```

>  - Mitre ATT&CK report, which displays the outcomes of ATT&CK techniques Monkey Agents tried to use.

Reviewer: ```suggestion
 - A MITRE ATT&CK report, which displays information about of the ATT&CK techniques that Monkey Agents tried to use.
```

>  - A Mitre ATT&CK report, which displays the information about the ATT&CK techniques that Monkey Agents tried to use.

Reviewer 2: ```suggestion
 - A MITRE ATT&CK report, which displays the information about the ATT&CK techniques that Monkey Agents tried to use.
```

>  - Zero Trust report, which displays violations of Zero Trust principles that Agents found.

Reviewer: ```suggestion
 - A Zero Trust report, which displays violations of Zero Trust principles that Monkey Agents found.
```

> More in depth description of reports generated can be found in [reports documentation page](/reports)

Reviewer: ```suggestion
A more in-depth description of these reports can be found in the [reports documentation page](/reports)
```
